mongodbatlas/atlas_checkpoints.go

```diff
@@ -0,0 +1,91 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	backupCheckpoints = "groups/%s/clusters/%s/backupCheckpoints"
+)
+
+// AtlasCheckpointService is an interface for interfacing with the Checkpoint
+// endpoints of the MongoDB Atlas API.
+type AtlasCheckpointService interface {
```

Comment:
Can we remove the `Atlas` part? This is already the atlas client so it should be clear enoght already